reefd/main.go

```diff
@@ -0,0 +1,124 @@
+package main
```

Comment:
you would like to apply `go fmt` to the file

Reply:
done